src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ThemeControllerCachingTests.cs

```diff
@@ -0,0 +1,98 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Cache;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Cache;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Controllers;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces.Cache;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
+using Moq;
+using NCrontab;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.Cache.CronSchedules;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
+using static GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils.MockUtils;
+using static Moq.MockBehavior;
+using static Newtonsoft.Json.JsonConvert;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Tests.Controllers;
+
+[Collection(CacheServiceTests)]
+public class ThemeControllerCachingTests : CacheServiceTestFixture
+{
+    private readonly IList<ThemeViewModel> _themes = ListOf(
+        new ThemeViewModel(Guid.NewGuid(), "slug1", "title1", "summary1"),
+        new ThemeViewModel(Guid.NewGuid(), "slug2", "title2", "summary2"));
+    
+    [Fact]
+    public async Task ListPublications_NoCachedEntryExists()
```

Comment:
All these method names are wrong - should be `ListThemes`

Reply:
Sorted ta